components/RandomFunction.js

```diff
@@ -61,7 +60,7 @@ function RandomFunction({ data }) {
         <StyledRandomButton onClick={handleRandomFriendClick}>
           <div>
             <h1> Be a friend! </h1>
-            <p>Push the Button!</p>
+            <p>Push the button!</p>
           </div>
         </StyledRandomButton>
```

Comment:
Here you do the came as above for you contact button. Try to solve this differently. And the h1 is not the correct element to use.